lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -36,11 +37,12 @@ def collect(cls, block_structure):
     def transform_block_filters(self, usage_info, block_structure):
         if not settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False):
             return [block_structure.create_universal_filter()]
+        # this needs to change ^^^
 
-        def is_proctored_exam_for_user(block_key):
+        def has_milestones_for_user(block_key):
             """
             Test whether the block is a proctored exam for the user in
-            question.
+            question, or requires any other unfulfilled milestones.
             """
             if (
                     block_key.block_type == 'sequential' and (
```

Comment:
If this is going to be applied to milestoned content that isn't a proctored exam, the `is_proctored_enabled` and `is_practice_exam` checks here are no longer needed, and that data doesn't need to be collected any longer either.